docs/2.Basics/Installing Checkov.md

```diff
@@ -20,6 +20,9 @@ pip3 install checkov
 
 ### Install on Alpine
 
+In general, it is not recommended to use Alpine with bigger Python projects, like `checkov`, due to the incompatibility of C extensions.
```

Comment:
```suggestion
In general, it is not recommended to use Alpine with larger Python projects, like Chekhov, because of incompatible C extensions.
Currently, Checkov can only be installed on Alpine with Python 3.11+, but it is not officially tested or supported.
```